tracer/build/_build/BuildVariables.cs

```diff
@@ -15,25 +15,34 @@ public static void AddContinuousProfilerEnvironmentVariables(this Dictionary<str
         envVars.AddTracerEnvironmentVariables(tracerHomeDirectory);
     }
 
-    public static void AddTracerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath tracerHomeDirectory)
+    public static void AddTracerEnvironmentVariables(this Dictionary<string, string> envVars, AbsolutePath monitoringHomeDirectory)
     {
         envVars.Add("COR_ENABLE_PROFILING", "1");
         envVars.Add("COR_PROFILER", "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}");
-        envVars.Add("COR_PROFILER_PATH_32", tracerHomeDirectory / "win-x86" / "Datadog.Trace.ClrProfiler.Native.dll");
-        envVars.Add("COR_PROFILER_PATH_64", tracerHomeDirectory / "win-x64" / "Datadog.Trace.ClrProfiler.Native.dll");
+
+        envVars.Add("COR_PROFILER_PATH_32", monitoringHomeDirectory / "Datadog.AutoInstrumentation.NativeLoader.x86.dll");
+        envVars.Add("COR_PROFILER_PATH_64", monitoringHomeDirectory / "Datadog.AutoInstrumentation.NativeLoader.x64.dll");
```

Comment:
maybe I'm getting confused again, but I thought `Datadog.Trace.ClrProfiler.Native.dll` was the native loader now 🤔

Reply:
yeah, this is the native loader for Linux only.
For windows, we need to change the layout (next PR)

Reply:
Urgh, we _really_ need to fix this asap 😩

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/2846